src/fswAlgorithms/imageProcessing/centerOfBrightness/centerOfBrightness.i

```diff
@@ -35,6 +35,7 @@ from Basilisk.architecture.swig_common_model import *
 %include "architecture/msgPayloadDefC/CameraImageMsgPayload.h"
```

Comment:
:small_red_triangle_down: Squash into the commit where you first add the .i if you like.

Reply:
Done